great_expectations/checkpoint/actions.py

```diff
@@ -409,9 +438,15 @@ def _run_pypd_alert(self, dedup_key: str, message: str, success: bool):
                 }
             )
 
-            return {"pagerduty_alert_result": "success"}
+            return ValidationActionResult(
+                status=ValidationActionRunStatus.SUCCESS,
+                run_info={"pagerduty_alert_result": "success"},
+            )
 
-        return {"pagerduty_alert_result": "none sent"}
+        return ValidationActionResult(
+            status=ValidationActionRunStatus.NOT_RUN,
+            run_info={"pagerduty_alert_result": "none sent"},
```

Comment:
For ms teams, the equivalent value is `None` instead of `"none sent"`. Can these be the same thing?

Reply:
Same note as above - don't love these dicts but doing what I can to minimize changes. If we can determine a consistent standard, I'd be happy to make changes across the board.